packages/discord.js/src/structures/Webhook.js

```diff
@@ -272,37 +269,23 @@ class Webhook {
   /**
    * Gets a message that was sent by this webhook.
    * @param {Snowflake|'@original'} message The id of the message to fetch
-   * @param {WebhookFetchMessageOptions|boolean} [cacheOrOptions={}] The options to provide to fetch the message.
-   * <warn>A **deprecated** boolean may be passed instead to specify whether to cache the message.</warn>
+   * @param {WebhookFetchMessageOptions} [options={}] The options to provide to fetch the message.
    * @returns {Promise<Message|APIMessage>} Returns the raw message data if the webhook was instantiated as a
    * {@link WebhookClient} or if the channel is uncached, otherwise a {@link Message} will be returned
    */
-  async fetchMessage(message, cacheOrOptions = { cache: true }) {
-    if (typeof cacheOrOptions === 'boolean') {
-      if (!deprecationEmittedForFetchMessage) {
-        process.emitWarning(
-          'Passing a boolean to cache the message in Webhook#fetchMessage is deprecated. Pass an object instead.',
-          'DeprecationWarning',
-        );
-
-        deprecationEmittedForFetchMessage = true;
-      }
-
-      cacheOrOptions = { cache: cacheOrOptions };
-    }
-
+  async fetchMessage(message, { cache = true, threadId } = { cache: true }) {
```

Comment:
empty object on the rhs, you're already defaulting

Reply:
```suggestion
  async fetchMessage(message, { cache = true, threadId } = {}) {
```